tests/storage/datavolume.go

```diff
@@ -807,6 +807,54 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 		})
 	})
 
+	Describe("[rfe_id:8400][crit:high][vendor:cnv-qe@redhat.com][level:system] Garbage collection of succeeded DV", func() {
+		var originalTTL *int32
+		ttl0 := int32(0)
+
+		BeforeEach(func() {
+			cdi := libstorage.GetCDI(virtClient)
+			originalTTL = cdi.Spec.Config.DataVolumeTTLSeconds
+		})
+
+		AfterEach(func() {
+			libstorage.SetDataVolumeGC(virtClient, originalTTL)
```

Comment:
I think you could try using ginkgo [defer](https://pkg.go.dev/github.com/onsi/ginkgo/v2#DeferCleanup) here

Reply:
why not AfterEach?

Reply:
no problems both are good, I think Defer might be easier to read in bigger tests, I am just pointing a new possibility here ;)